modules/roller/templates/dev/env.erb

```diff
@@ -5,35 +5,36 @@ PYTHONDONTWRITEBYTECODE=1
 DJANGO_CONFIGURATION=Dev
 DJANGO_SECRET_KEY=<%= @django_secret_key %>
 
+CELERY_TASK_SOFT_TIME_LIMIT=2000
+CELERY_TASK_TIME_LIMIT=3000
+
 # Only override this if you need something different from the default
 #TIME_ZONE=UTC
 #CONN_MAX_AGE=60
 ALLOWED_HOSTS=*
 CORS_ORIGIN=*
 USE_X_FORWARDED_HOST=True
 
-#REDIS_URL=redis://redis:6379/0
-
 # Get these settings from Taskcluster
 TASKCLUSTER_CLIENT_ID=<%= @taskcluster_client_id %>
 TASKCLUSTER_ACCESS_TOKEN=<%= @taskcluster_access_token %>
 
 REQUIRED_TASKCLUSTER_SCOPE_SETS=project:releng:roller-dev:*
 TASK_NAMES=ping,ipmi_list,ipmi_status,ipmi_reset,ipmi_cycle,ipmi_off,ipmi_on,reboot,file_bugzilla_bug
+REBOOT_METHODS=ssh_reboot,snmp_rebootdelay,ipmi_reset,ipmi_cycle,ipmi_on,file_bugzilla_bug
 
-VALID_WORKER_ID_REGEX=^.*(ms|yosemite)-.*
-
-NOTIFY_EMAIL=relops-hardware-controller@mozilla.com
+VALID_WORKER_ID_REGEX=^.*(MS|ms|yosemite)-.*
 
-# RelOps HW Creds
 BUGZILLA_API_KEY=<%= @bugzilla_api_key %>
 BUGZILLA_URL=https://bugzilla-dev.allizom.org
 BUGZILLA_REOPEN_STATE=UNCONFIRMED
 # Change bug templates from default because bugzilla-dev does not have the "CIDuty" component
-BUGZILLA_WORKER_TRACKER_TEMPLATE={"product": "Infrastructure & Operations", "component": "RelOps", "summary": "${hostname} problem tracking", "alias": "${alias}", "version": "unspecified", "assigned_to": "NOTIFY_EMAIL", "api_key": "${api_key}"}
+BUGZILLA_WORKER_TRACKER_TEMPLATE={"product": "Infrastructure & Operations", "component": "RelOps", "summary": "[${DC}] ${alias} problem tracking", "alias": "${alias}", "version": "unspecified", "assigned_to": "NOTIFY_EMAIL", "api_key": "${api_key}"}
```

Comment:
`"alias": "${alias}"` - Won't this fail if we already have a bug with an existing alias? 
A lot of them are locked with alias in CiDuty's bugzilla component.

Reply:
This is going over to bugzilla-dev. On there, there is no CIDuty and so I have it going to RelOps for testing.

Reply:
Thanks for the clarification!